testing/dart/window_test.dart

```diff
@@ -22,8 +22,8 @@ void main() {
   });
 
   test('FrameTiming.toString has the correct format', () {
-    final FrameTiming timing = FrameTiming(<int>[1000, 8000, 9000, 19500]);
-    expect(timing.toString(), 'FrameTiming(buildDuration: 7.0ms, rasterDuration: 10.5ms, totalSpan: 18.5ms)');
+    final FrameTiming timing = FrameTiming(<int>[500, 1000, 8000, 9000, 19500]);
+    expect(timing.toString(), 'FrameTiming(buildDuration: 6.5ms, rasterDuration: 10.5ms, vsyncOverhead: 0.5ms, totalSpan: 18.5ms)');
```

Comment:
I still expect the `buildDuration` to be 7ms. I can also probably understand if it's 7.5ms (8000 - 500). But I'm not sure how this 6.5ms is calculated?

Reply:
Done. should be 7ms.